testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/client/CamundaApiClient.java

```diff
@@ -100,6 +102,14 @@ public ProcessInstanceDto getProcessInstanceByKey(final long processInstanceKey)
     return objectMapper.readValue(body, ProcessInstanceDto.class);
   }
 
+  public SearchProcessInstanceResponseDto findProcessInstances() throws IOException {
+    ensureAuthenticated();
+
+    final String requestBody = "{}";
+    final String responseBody = sendPostRequest(PROCESS_INSTANCE_SEARCH_ENDPOINT, requestBody);
+    return objectMapper.readValue(responseBody, SearchProcessInstanceResponseDto.class);
```

Comment:
❓ Why did you map the response into another DTO object ? Can we re-use the one available from the client ? In this case will be `ProcessInstanceSearchQueryResponse`

Reply:
At the moment, CPT invokes the v1 Operate API. The client DTOs are part of the v2 API.

When we switch to the new API and use the SDK, we can remove the DTOs from CPT. The new `SearchProcessInstanceResponseDto` uses the already existing `ProcessInstanceDto`. 

For now, let's keep the internal CPT DTOs.

Reply:
Yep, sorry for it